### PR TITLE
fix: format pretty code and run husky when start project

### PR DESCRIPTION
**Pull request by huydosgtech**

### CODE REVIEW

File: template/scripts/post-install.sh [hunk 1]
```diff
@@ -7,9 +7,13 @@ if [[ ! "$CI" = true ]]; then
   if [[ $(sysctl -n machdep.cpu.brand_string) =~ "Apple" ]]; then
     echo "M1"
     cd ./ios && arch -x86_64 npx pod install && cd ..
+    echo "prepare husky"
+    yarn prepare
   else
     echo "Intel"
     cd ./ios && npx pod install && cd ..
+    echo "prepare husky"
+    yarn prepare
```
Comment (loc-nguyenthien) [inline template/scripts/post-install.sh:16]: Remove husky command

File: template/.husky/pre-commit [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
+cd template
```
Comment (loc-nguyenthien) [inline template/.husky/pre-commit:4]: Remove this

**Comment by ThinhKimVo** [comment]
move husky out root

**Comment by ThinhKimVo** [comment]
resolve conflict please

**Pull request closed without merging**